src/hooks/usePanResponder.ts

```diff
@@ -22,6 +22,7 @@ import {
   getDistanceBetweenTouches,
   getImageTranslate,
   getImageDimensionsByTranslate,
+  getImageTransform,
```

Comment:
Hey @LimitedDani thanks for this PR, can I ask you what its import for as its never used below?

Reply:
Ummm, i dont know, i will remove it

Reply:
@antonKalinin It is fixed